### ISSUE TITLE
Excel issue needs option to force delimeter.wrap

### ISSUE THREAD

**Issue by peacechen** [mrodrig/json-2-csv#188]
Excel automatically converts `false` boolean types to `FALSE`. Users who load and edit the CSV in Excel end up with booleans values that are uppercase `TRUE/FALSE` instead of lowercase. This does not parse back properly with `csv2json`. If I manually edit the CSV and wrap the boolean true/false in quotes, it remains lowercase after editing in Excel.

The `delimeter.wrap` option has no effect for all the types that I've tried. This feature had been requested in #133 but wasn't implemented.

The alternative would be to modify `csv2json` to be able to parse back TRUE/FALSE boolean values. That would be messier though.

## Background Information
- Module Version: `3.10.3`
- Node/Browser Version: `Chrome 86`

The issue I'm reporting is with:
- [*] json2csv
- [ ] csv2json

I have...
- [*] searched to see if an issue has already been reported.
- [*] verified that my JSON/CSV data is valid (using something like http://jsonlint.com or https://csvlint.io/).
- [*] tried upgrading to the latest version of json-2-csv (since the issue may already be fixed).

## Expected Behavior
"true", "false"

## Actual Behavior
true, false

## Data Sample 
CSV:
```csv
MY_BOOL,FIELD1,FIELD2
true,data1,data2
true,data3,data4
```

## Code Example
```javascript
// Please include a simple example to replicate the issue
let converter = require('json-2-csv');

json2csv(data, onCompleted, {
  delimiter: {
    wrap: "\"",
  },
});

```

**Comment by mrodrig** [comment]
Thanks for reporting this @peacechen. I can definitely see the use case for forcing boolean values to be wrapped since Excel is interpreting them when not wrapped in quotes. I've added a `wrapBooleans` options that will force these Boolean values to be wrapped in the `options.delimiter.wrap` value and released it in version 3.12.0 just now. Hope this helps.

**Comment by peacechen** [comment]
Thanks for the fast turn-around!

A small TS issue: passing in `wrapBooleans` to options results in an error

> Argument of type '{ excludeKeys: string[]; wrapBooleans: boolean; }' is not assignable to parameter of type 'IFullOptions'.
>   Object literal may only specify known properties, and 'wrapBooleans' does not exist in type 'IFullOptions'.ts(2345)

It looks like `wrapBooleans` needs to be added to the TS definition for `IFullOptions`.
https://github.com/mrodrig/json-2-csv/blob/258f048312936d3897f2d5a43b47f7df1bf62db0/lib/converter.d.ts#L45

**Comment by mrodrig** [comment]
Sorry about that - I'll merge your PR and will get that released in `3.13.0` in a few minutes. Thanks for the PR!

**Comment by peacechen** [comment]
Thanks again for the fast merge and release 😄
